onadata/apps/logger/models/instance.py

```diff
@@ -443,7 +446,9 @@ def get_notes(self):
 
     def get_review_status_and_comment(self):
         """
-        Return a tuple of review status and comment
+        Return a tuple of review status and comment.
+        Deprecated in favour of `get_latest_review`
+        TODO: Clean code of this unused function.
```

Comment:
I wonder if we should use something like the [Deprecated](https://pypi.org/project/Deprecated/) library or a [custom decorator](https://stackoverflow.com/a/30253848) like this; to mark deprecated functions ?

_TODOs usually get forgotted IMO_

Reply:
Thanks for this.
I've used the pip deprecated package for this. Seemed cleaner.